todos-api/Dockerfile

```diff
@@ -5,8 +5,8 @@ EXPOSE 8082
 WORKDIR /usr/src/app
 
 COPY package-lock.json package.json *.js /usr/src/app/
-COPY routes /usr/src/app/routes
+COPY routes.js /usr/src/app/routes
```

Comment:
Better just remove this line, because previous one copies all `*.js`-files already.
This line remains here because initially all the routes were in a directory, but after refactoring I kept using old image 🤦‍♂️

Reply:
Fixed this in my master to "stop the bleeding". Please rebase your branch